hosts

```diff
@@ -36,6 +36,8 @@ lnlsfac-srv1 # (10.128.255.9, 10.0.38.83)  # (linac server)
 [nfs_servers]
 lnls454-linux
 
+[tmp_group]
```

Comment:
Isn't it better to name it as something like `server_web`?

Reply:
oh, this is a tmp host group, for testing... it is not a group for web servers.